README.md

```diff
@@ -1,4 +1,121 @@
-RZMessagingWindow
-=================
+# RaisinToast
 
-A UIWindow subclass used to message information to the users of your app.
+[![CI Status](http://img.shields.io/travis/adamhrz/RaisinToast.svg?style=flat)](https://travis-ci.org/adamhrz/RaisinToast)
+[![Version](https://img.shields.io/cocoapods/v/RaisinToast.svg?style=flat)](http://cocoadocs.org/docsets/RaisinToast)
+[![License](https://img.shields.io/cocoapods/l/RaisinToast.svg?style=flat)](http://cocoadocs.org/docsets/RaisinToast)
+[![Platform](https://img.shields.io/cocoapods/p/RaisinToast.svg?style=flat)](http://cocoadocs.org/docsets/RaisinToast)
+
+RaisinToast provides a messaging window layer and a default "toast" view controller, ideal for presenting errors, warnings and feedback throughout your app.
+
+![RaisinToast in action](https://github.com/Raizlabs/RaisinToast/blob/feature/ahowitt/initial-refactor/Example/raisin-toast-6plus.gif "RaisinToast Demo Project")
+
+## Installation
+
+RaisinToast is available through [CocoaPods](http://cocoapods.org). To install
+it, simply add the following line to your Podfile:
+
+    pod "RaisinToast"
+
+
+## Demo Project
+
+A demo project is available in the Example directory. The demo project uses CocoaPods, and can be opened from a temporary directory by running
+
+    pod try RaisinToast
+
+Alternatively, the demo can be configured by running the following commands from the root project directory.
+
+    cd Example
+    pod install
```

Comment:
I thought we decided not to use pods in the example project to allow users not using cocoa pods to run it?